cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -222,7 +227,14 @@ func buildPartitionList(ptable *sfdiskPartitionTable, pv *gadget.LaidOutVolume)
 		fmt.Fprintf(buf, "%s : start=%12d, size=%12d, type=%s, name=%q\n", node, p.StartOffset/sectorSize,
 			s.Size/sectorSize, partitionType(ptable.Label, p.Type), s.Name)
 
-		// Are roles unique so we can use it to map nodes? Should we use labels instead?
+		// Set expected labels based on role
+		switch s.Role {
+		case systemBootRole:
```

Comment:
We could use the role names defined in the gadget package. i.e.:
```
diff --git a/cmd/snap-bootstrap/partition/sfdisk.go b/cmd/snap-bootstrap/partition/sfdisk.go
index fd80b7b0ef..7347362905 100644
--- a/cmd/snap-bootstrap/partition/sfdisk.go
+++ b/cmd/snap-bootstrap/partition/sfdisk.go
@@ -31,8 +31,6 @@ import (
 )
 
 const (
-       systemBootRole  = "system-boot"
-       systemDataRole  = "system-data"
        ubuntuBootLabel = "ubuntu-boot"
        ubuntuDataLabel = "ubuntu-data"
 
@@ -229,9 +227,9 @@ func buildPartitionList(ptable *sfdiskPartitionTable, pv *gadget.LaidOutVolume)
 
                // Set expected labels based on role
                switch s.Role {
-               case systemBootRole:
+               case gadget.SystemBoot:
                        s.Label = ubuntuBootLabel
-               case systemDataRole:
+               case gadget.SystemData:
                        s.Label = ubuntuDataLabel
                }
 
```

Reply:
Using the names already defined is much better, thanks.